image/system_a/data/lang/Deutsch.json

```diff
@@ -338,6 +338,7 @@
   "app_call_wrong_emergency": "Anruf nicht möglich.\n$NUMBER ist keine Notrufnummer.",
   "app_messages_title_main": "Nachrichten",
   "app_messages_no_messages": "<text align='center' color='4'>Noch keine Nachrichten.<p>Zum Hinzufügen <b>linken Pfeil</b> drücken.</p></text>",
+  "app_messages_no_templates": "<text align='center'>Noch keine Vorlagen.<p><b>Erstellen Sie eine neue Vorlage</b></p><p>im Mudita Center, um sie in Ihrem</p><p>Pure zu verwenden.</p></text>",
```

Comment:
Consult the design team if the new text is suitably laid out in various languages regarding line wrapping.

Reply:
Confirmed that the layouts are OK.